### ISSUE TITLE
CPU usage even when not playing ESO

### ISSUE THREAD

**Issue by tcumbow** [Phrosty1/PixelsToKeys#1]
The CPU usage isn't too bad, but it is enough where I want to terminate PTK when not playing ESO.

**Comment by tcumbow** [comment]
I see two options:
#1 - Compile PTK using Java 9 or later so that we can call some code that checks if "eso64" is running. Technically, we could still do this with Java 8, but it involves calling a command line tool and parsing the output 🤮
#2 - Write a separate app or script that watches for "eso64" and automatically starts/stops PTK

@Phrosty1 Which options sounds better to you?

**Comment by Phrosty1** [comment]
I'd like you to go with option 1! That parsing thing rubbed me the wrong way, then I forgot about it.

**Comment by Phrosty1** [comment]
I'm taking a crack at doing this. I haven't compiled java in a while so it may not be smooth.

**Comment by tcumbow** [comment]
@Phrosty1 
I've identified another possible (and better) solution.  Checking if eso64.exe isn't really the best way to get what we want.  What we really want is to know if ESO is the active window.  This could be accomplished in Java 8, so we wouldn't have to mess with upgrading.

https://sl.bing.net/jKOWkFhjcHY

**Comment by Phrosty1** [comment]
Yup, that's what I thought we were trying to do but I might have missed a conversation.
Sidenote, things shared from bing are just links to bing.com without any context. It's really fucking dumb.
